lisa/sut_orchestrator/azure/arm_template.json

```diff
@@ -472,12 +472,16 @@
                         {
                             "name": "publicKeyData",
                             "type": "string"
+                        },
+                        {
+                            "name": "disable_password_authentication",
```

Comment:
Does it impact password only or password first scenario, like Windows?

Reply:
This is in fun getLinuxConfiguration, so it is only for Linux.  In fun getOsProfile, only when the key data is not empty and is Linux, then it calls getLinuxConfiguration fun.

So this won't impact the password only or the password first scenario.